### DISCUSSION TITLE
Error: ⚠️Imgur upload failed, check dev console

### DISCUSSION THREAD

**Discussion by thammawat-s** [gavvvr/obsidian-imgur-plugin#82] [Getting help]
Hello I ran into this issue when upload.

![image](https://github.com/user-attachments/assets/fec9c1fb-d978-4cf7-b933-b505f05a03cc)

**Comment by gavvvr** [comment]
Hello @thammawat-s 
Please upgrade to the latest version (2.6.0), and add your personal Client ID at Imgur plug-in settings tab. You will find all necesary instructions right in plugin's settings tab. Please also don't forget to re-authenticate after providing personal Client ID. Otherwise, the plugin will continue using an old session, which is tied to old problematic Client ID (used by all plugin users before 2.6.0)

**Reply by thammawat-s** [reply]
Thank you @gavvvr .  I revoke old access and  re-authenticate per instruction in "Getting started" section.   This now works wonderful.  Thank you very much!
